core/modules/qproc/QuerySession.h

```diff
@@ -145,6 +144,16 @@ class QuerySession {
     explicit QuerySession(Test& t); ///< Debug constructor
     std::shared_ptr<query::QueryContext> dbgGetContext() { return _context; }
 
+    /**
+     *  Print query session to stream.
+     *
+     *  Used for debugging
+     *
+     *  @params out:    stream to update
+     *  @return:        out
+     */
+    std::ostream& print(std::ostream& out) const;
```

Comment:
Returning `std::ostream&` from this method is not very useful, it's not going to be used in the same way as `operator<<()`. I usually define that sort of methods as returning `void`.